graphene/types/enum.py

```diff
@@ -67,6 +67,28 @@ def from_enum(cls, enum, description=None, deprecation_reason=None):  # noqa: N8
 
 
 class Enum(six.with_metaclass(EnumMeta, UnmountedType, BaseType)):
+    """
+    Enum type defintion
+
+    Defines a static set of values that can be provided as a Field, Argument or InputField.
+
+    .. code:: python
+
+        class NameFormat(graphene.Enum):
```

Comment:
```suggestion
        class NameFormat(Enum):
```

Reply:
I think it's better to keep graphene to show exactly which `Enum` we mean.

Reply:
This may be worth a larger conversation. Naming collisions with python modules like enum.Enum, for example. Perhaps a naming protocol going forward?

As @ProjectCheshire pointed out in https://graphenetools.slack.com/archives/CGR4VCHUL/p1559524808014100, core-next has these types available as, for example:

```
from graphql import (
    GraphQLSchema, GraphQLObjectType, GraphQLField, GraphQLString)
```

See https://graphql-core-next.readthedocs.io/en/latest/intro.html#getting-started

Reply:
I'll keep the import close in all examples to avoid confusion.

Agree with @phalt that the clash for name `Enum` is a little confusing. I do prefer the shorter names in code as these objects are used very often in the schema. `GraphQL` as a prefix seems like overkill in most cases (especially given the namespace `graphql` and `graphene` for the module).